### ISSUE TITLE
4.0 beta 2 not loading

### ISSUE THREAD

**Issue by kger** [cspotcode/virtual-desktop-grid-switcher#18]
When I double-click on it to run it, something happens because the screen flashes but there is no tray icon added.

**Comment by mzomparelli** [comment]
@kger, Try my app. I think you'll like it better than this.

https://github.com/mzomparelli/zVirtualDesktop

**Comment by cspotcode** [comment]
Yeah, my project is broken by changes in Windows and needs an update.  The Window Virtual Desktop API changed, so I have to rebuild with the new version.  I also want to convert this to an AutoHotKey-centric project.  Basically it'll be a DLL that's loaded by an AHK script, meaning you can use all of AHK's features for custom hotkeys and automation.

@mzomparelli's project looks cool; I hope it has what you need.

**Comment by mzomparelli** [comment]
@cspotcode  that's exactly why I created zVirtualDesktop. Everyone seems to be making this for AHK. In my opinion, that is too much for your average user. I love AHK, but I'm a tech savvy though. I think zVirtualDesktop handles everything you need for Windows 10 Virtual Desktops and handles it well. I appreciate you mentioning it and saying it looks cool. It's a very handy app.
